src/main/scala/cromwell/backend/BackendValidationActor.scala

```diff
@@ -0,0 +1,50 @@
+package cromwell.backend
+
+import akka.actor.{Actor, ActorLogging}
+import akka.event.LoggingReceive
+import cromwell.backend.BackendValidationActor.{Validate, ValidationResult}
+import wdl4s.NamespaceWithWorkflow
+
+import scala.concurrent.Future
+
+object BackendValidationActor {
+  sealed trait BackendValidationActorMessage
+  case class Validate(namespace: NamespaceWithWorkflow, wfOptionsJson: String, wfInputsJson: Option[String] = None) extends BackendValidationActorMessage
+  case class ValidationResult(isSuccess: Boolean) extends BackendValidationActorMessage
```

Comment:
I think this would be better as an ADT

```scala
sealed trait WorkflowValidationResult
final case class WorkflowValidationFailure(failures: ErrorOr[Unit]) extends WorkflowValidationResult
case object WorkflowValidationSuccess
```

Though some may quibble with the `ErrorOr`... :smile: